### PR TITLE
Use :class:`~chainer.Variable` in RNNs docs

### PR DESCRIPTION
**Pull request by crcrpar**
Fix the link to `chainer.Variable` in chainer/(functions | links)/n_step_(rnn | lstm | gru).py.

### CODE REVIEW

File: chainer/functions/connection/n_step_gru.py [hunk 1]
```diff
@@ -63,24 +63,26 @@ def n_step_gru(
     Args:
         n_layers(int): Number of layers.
         dropout_ratio(float): Dropout ratio.
-        hx (chainer.Variable): Variable holding stacked hidden states.
+        hx (:class:`~chainer.Variable`):
```
Comment (toslunar) [inline chainer/functions/connection/n_step_gru.py]:
```suggestion
        hx (~chainer.Variable):
```

Comment (crcrpar) [reply chainer/functions/connection/n_step_gru.py]: Thank you!

**Comment by Crissman** [comment]
Jenkins, test this please.

**Comment by Crissman** [comment]
LGTM. Merging!

**Pull request merged**